src/components/elementView/elementTabContainer.tsx

```diff
@@ -40,7 +40,12 @@ class ElementTabContainer extends React.Component<
         />
 
         <Tab className="tab" id="rack" title="Racks" panel={<RackTab />} />
-        <Tab className="tab" id="datacenter" title="Datacenters" />
+        <Tab
+          className="tab"
+          id="datacenter"
+          title="Datacenters"
+          panel={<ElementTab {...this.props} element={ElementType.DATACENTER} />}
+        />
```

Comment:
we should hide this from the user if the user is not admin, so there should be some logic for that here